packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -374,12 +373,26 @@ export const contract = async (
 
     const resolverHandler = (seat: ZCFSeat) => {
       seat.exit();
+      // Keep original pattern for backward compatibility  
       return harden({ invitationMakers: makeResolverInvitationMakers });
     };
 
     return zcf.makeInvitation(resolverHandler, 'resolver', undefined);
   };
 
+  // New invitation that returns the service directly for invokeEntry support
```

Comment:
If a function isn't sufficiently self-documenting, use a `/** ... */` docstring to document it. `//` comments are for implementation details, not for API docs.

Also: write it in the ever-present. New vs. old makes sense in the context of a change log, but not in the resulting code.